docs/security-considerations.rst

```diff
@@ -63,13 +63,13 @@ complete contract):
       mapping(address => uint) shares;
       /// Withdraw your share.
       function withdraw() {
-          if (msg.sender.send(shares[msg.sender]))
+          if (msg.sender.call.value(shares[msg.sender])())
               shares[msg.sender] = 0;
       }
   }
 
 The problem is not too serious here because of the limited gas as part
-of ``send``, but it still exposes a weakness: Ether transfer always
+of ``call``, but it still exposes a weakness: Ether transfer always
```

Comment:
Please see the first part of this sentence "not too serious here because of the limited gas".

I think it would be a good idea to leave the above sentence as it is and introduce a new example using `call` and explain that is even more dangerous than `send`.

What do yo think?

Reply:
I agree. I added a new example that uses call. Please review again.